scripts/prohibit-java-lines-split-needlessly

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+set -e -o pipefail -u
+
+self_dir=$(cd "$(dirname "$0")" &>/dev/null; pwd -P)
+source "$self_dir"/utils.sh
+
+awk_script="$(cat <<'EOF'
+  function indent_width(str) {
```

Comment:
POSIX `awk` is awesome for doing stuff **between* lines, but surprisingly weak when processing a single line... I don't think there's an easy standard (non-GNU-specific) way to do stuff like this here